tracing/src/main/java/io/micronaut/tracing/jaeger/JaegerConfiguration.java

```diff
@@ -277,4 +289,56 @@ public void setSenderConfiguration(@Nullable JaegerSenderConfiguration senderCon
             return configuration;
         }
     }
+
+    /**
+     * The codec configuration bean
+     * @author Manuel Martin
+     * @author Burt Beckwith
+     */
+    @ConfigurationProperties("codec")
```

Comment:
I don't think it's necessary to add a new configuration property class that encapsulates just one string field. Is there a problem to have the `codec` (`codecs`) to be part of `JaegerConfiguration` ?